Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:22.04
```

Comment:
probably macos specific but I get an error trying to locate `lib32z1` 

adding a `--platform=linux/amd64` seemed to fix it, might be good to add it to readme or something.

(can confirm its reproducible on my end, removing that option causes it to fail again) 

still building right now but fyi @oliveross

Reply:
Ah, good catch, I hadn't realized it would be using a different architecture by default on Mac. I just added the platform to the dockerfile, since I'm pretty sure we always want to be targeting `linux/amd64`.

Reply:
other than that I think it seems to be working, the fetch command seems to be taking a while but I'm assuming that's just par for the course

Reply:
it's also taking a while to build but it seems to be working

Reply:
how long does it usually take?

Reply:
yeah it's probably cause of the virtualization. I'm still on the build step for the past like 3 hours now but I'm assuming it's still chugging along cause the log file keeps getting more output. My log is here: 

[build_rj4jvjb3.log](https://github.com/ecobee/webrtc-android/files/13505677/build_rj4jvjb3.log)

Reply:
Damn that's a pretty massive performance drop still. I do have a 32-thread CPU, which might be another part of the difference. Like you say, it still seems to be going, so I guess just give it more time.

Reply:
yeah i'll just leave this running and i'll update tomorrow with the stats on it as a sort of run time estimate that we can add to this so people know to expect a long build time due to virtualization. 

So far for the fetch step: 
```
> stat build_ap9lvcq7.log
Access: 2023-11-29 16:37:08.376224002 +0000
Modify: 2023-11-29 17:49:29.573342012 +0000
Change: 2023-11-29 17:49:29.573342012 +0000
 Birth: 2023-11-29 16:37:08.376224002 +0000
```
so I think from birth - modify fetch took about an hour for me.

Reply:
Access: 2023-11-29 21:32:12.889476003 +0000
Modify: 2023-11-30 04:07:51.569905007 +0000
Change: 2023-11-30 04:07:51.569905007 +0000
 Birth: 2023-11-29 18:25:27.410375011 +0000

lol